docs/development/manufacturer/manufacturer-design-guidelines.md

```diff
@@ -294,6 +294,10 @@ For details of the use of these LEDs, please see the [FC LEDs](/docs/development
 
 ## 3.2 Resource Selection Considerations
 
+- Betaflight does not support sharing devices on the SPI bus which is blocking excution and results in bad performance. Mainly sharing MAX7456 and blackbox generates support issues.
+- Optimal motor assignment would use the same channel each four motors requiring only one DMA channel used for dshot telemetry with bitbang enabled.
```

Comment:
I don't really understand what this means. Can we split this up into multiple sentences?

Reply:
Agree - a bit too cryptic. Updated.